src/Umbraco.Core/Migrations/Install/DatabaseDataCreator.cs

```diff
@@ -210,12 +210,12 @@ private void CreatePropertyTypeGroupData()
 
         private void CreatePropertyTypeData()
         {
-            _database.Insert(Constants.DatabaseSchema.Tables.PropertyType, "id", false, new PropertyTypeDto { Id = 6, UniqueId = 6.ToGuid(), DataTypeId = 1043, ContentTypeId = 1032, PropertyTypeGroupId = 3, Alias = Constants.Conventions.Media.File, Name = "Upload image", SortOrder = 0, Mandatory = false, ValidationRegExp = null, Description = null, Variations = (byte) ContentVariation.Nothing });
+            _database.Insert(Constants.DatabaseSchema.Tables.PropertyType, "id", false, new PropertyTypeDto { Id = 6, UniqueId = 6.ToGuid(), DataTypeId = 1043, ContentTypeId = 1032, PropertyTypeGroupId = 3, Alias = Constants.Conventions.Media.File, Name = "Upload image", SortOrder = 0, Mandatory = true, ValidationRegExp = null, Description = null, Variations = (byte) ContentVariation.Nothing });
```

Comment:
@bergmania do we need a migration for this - so that existing V8 sites we are all running make these properties mandatory on Image & File media types?

Reply:
I has in doubt, but ended up not doing it, for the same reason as we did not do it for https://github.com/umbraco/Umbraco-CMS/issues/4322